test/OpenTelemetry.Contrib.Instrumentation.Runtime.Tests/RuntimeMetricsTests.cs

```diff
@@ -54,5 +54,27 @@ public void RuntimeMetricsAreCaptured()
             var metric1 = exportedItems[0];
             Assert.StartsWith(MetricPrefix, metric1.Name);
         }
+
+        [Fact]
+        public async Task CpuUsageMetricAreCaptured()
+        {
+            var exportedItems = new List<Metric>();
+            var metricReader = new PeriodicExportingMetricReader(new InMemoryExporter<Metric>(exportedItems), 500);
+
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                 .AddRuntimeMetrics(options =>
+                 {
+                     options.CpuEnabled = true;
+                 })
+                 .AddReader(metricReader)
+                .Build();
+
+            await Task.Delay(1000);
```

Comment:
this is not required, right?

Reply:
```
var afterTenMsec = DateTime.UtcNow.AddMilliseconds(10);
while (DateTime.UtcNow < afterTenMsec) { }
```

Can we do something like above, and forceflush, and validate that the value is >=10 ?

Reply:
Something like this
``` csharp
var afterTenMsec = DateTime.UtcNow.AddMilliseconds(10);
            while (DateTime.UtcNow < afterTenMsec)
            {
                meterProvider.ForceFlush(MaxTimeToAllowForFlush);
            }

            Assert.True(exportedItems.Count > 10);
```

Reply:
Not really..`while (DateTime.UtcNow < afterTenMsec) { }` is intentionally spinning CPU.
Flush after that.
There'll be single exported item.
But its value should be atleast 10.